pkg/config/constants.go

```diff
@@ -46,6 +46,11 @@ const (
 	// WorkspaceNameLabel is label key to store workspace name
 	WorkspaceNameLabel = "controller.devfile.io/workspace_name"
 
+	// PullSecretLabel marks the intention that secret should be used as pull secret for devworkspaces withing namespace
+	// Only secrets with 'true' value will be mount as pull secret
+	// Should be assigned to secrets with type docker config types (kubernetes.io/dockercfg and kubernetes.io/dockerconfigjson)
+	DevWorkspacePullSecretLabel = "controller.devfile.io/workspace_pull_secret"
```

Comment:
Here we go with naming again :)

Should this be `devworkspace_pull_secret`?

Reply:
yes and no. 

If yes, we must change `controller.devfile.io/workspace_name` and `controller.devfile.io/workspace_id` which probably makes sense.

I'll do it

Reply:
changed to devworkspace_pull_secret. workspace_id and name will be changed in another PR.

maybe even devworkspace_pullsecret would be even better.

Reply:
we agreed and now it's changed to `controller.devfile.io/deworkspace_pullsecret`